test/conformance/api/shared/util.go

```diff
@@ -118,17 +118,18 @@ func checkResponses(t pkgTest.TLegacy, num, min int, domain string, expectedResp
 	//   WHERE body IN $expectedResponses
 	//   GROUP BY body
 	// )
-	for _, ar := range actualResponses {
+	for i, ar := range actualResponses {
 		if er := substrInList(ar, expectedResponses); er != "" {
 			counts[er]++
 		} else {
 			badCounts[ar]++
+			t.Logf("For domain %s: got unexpected response for request %d", domain, i)
```

Comment:
```suggestion
			t.Logf("domain %s: got unexpected response for request %d", domain, i)
```

nit, to be consistent with the logs below.

Reply:
Actually, there are now 3 `t.Logf` which start with "For domain" (I added just one new, there were other two examples). I believe it's because it's t.Log vs. t.Errorf - the first one is usually at the beginning of the printed message and the second one can be buried down in the logs. I'd just keep it as it is. Thanks

Reply:
wfm, thanks!